src/fastoad/io/tests/test_data_file.py

```diff
@@ -76,6 +80,16 @@ def test_datafile_save_read(cleanup, variables_ref):
 
     assert set(data_file_2) == set(variables_ref)
 
+    # Check using text file object --------------------
```

Comment:
I see that comments have been put on those part of the test to explain what is done, but there are no comment on the rest of the test. It may be worth adding